<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DealWithSpacesDialog</class>
 <widget class="QDialog" name="DealWithSpacesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>You've entered spaces</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The text you entered contains spaces. What would you like to do?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="splitWord">
     <property name="text">
      <string>Split the word, using the spaces as a guide</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="convertSpaces">
     <property name="text">
      <string>Convert the spaces to non-breaking spaces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="cancelEdit">
     <property name="text">
      <string>Cancel the edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
